_includes/meta.html

```diff
@@ -1,6 +1,5 @@
 <meta charset="utf-8" />
 <meta content="width=device-width, initial-scale=1" name="viewport" />
-<meta content="IE=edge" http-equiv="X-UA-Compatible" />
```

Comment:
would it make sense for us to display the IE banner on this site? or not worth drawing attention to? We could use the `<!-- if IE -->` to render the banner statically, right?